tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java

```diff
@@ -1025,32 +1023,6 @@ public String getDescription(String path, String value, Object context) {
         return description;
     }
 
-    /**
-     * Creates an escaped HTML string of placeholder information.
-     *
-     * @param path the xpath to specify placeholder information for
-     * @return a HTML string, or an empty string if there was no placeholder information
-     */
-    public String getPlaceholderDescription(String path) {
```

Comment:
huh, i wonder when this got dropped?  Probably accidental.

This was added in CLDR-4544, and then unknowingly replicated in https://github.com/unicode-org/cldr/pull/2012